flocker/control/test/test_protocol.py

```diff
@@ -563,6 +563,7 @@ def test_nodestate_notifies_all_connected(self):
             self.client.callRemote(NodeStateCommand,
                                    state_changes=(NODE_STATE,),
                                    eliot_context=TEST_ACTION))
+        self.reactor.advance(10.0)
```

Comment:
:question: Then this could be `advance(BROADCAST_DELAY + 1)` which might be clearer.